notebooks/climatiq.ipynb

```diff
@@ -335,6 +436,48 @@
     "\n"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Industry classification estimate\n",
+    "\n",
+    "If your data comes with an industry specification code Climatiq API allows you to query emission factors, based on their NACE or ISIC industry classification codes. At the momemnt this endpoint works only with spend-based datasets, in particular EXIOBASE and GHG Protocol. \n",
+    "[More details about the industry classification estimate endpoint here](https://www.climatiq.io/docs#classifications) "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "API_token = \"your token\" # add your Climatiq API token here\n",
+    "url = \"https://beta3.api.climatiq.io/classifications/estimate\"\n",
+    "authorization_header = {\"Authorization\": \"Bearer \"+ API_token} # this is an authorization header which allows API to recognize you as a valid user\n",
+    "\n",
+    "\n",
+    " data = {\n",
+    "            \"classification\": {\n",
+    "            \"classification_type\": \"nace2\", # here add your classification type, at the momement it will be either nace2 or isic4\n",
+    "            \"classification_code\": \"nace_code\", # here add a code from your classification, e.g. NACE 1 - \n",
+    "            \"source\": \"EXIOBASE\", # choose the desired source - at them momement, only EXIOBASE and GHG Protocol data are mapped to the industry classification codes\n",
+    "             \"year\": \"2021\", # choose year of interest\n",
+    "            \"region\": \"DE\" # choose location of interest, double digit UN Locode or one of the pre-defined location codes - global from GHG Protocl or one of the 5 pre-defined \n",
+    "            # \"rest of the world locations\" from EXIOBASE, see documentation around OEFDB for more infromation\n",
+    "        },\n",
+    "        \"parameters\": {\n",
+    "            \"money\": 3000,  # provide amount of money spend on the specific activity\n",
+    "            \"money_unit\": \"usd\" # choose currency, currently we support more than 30 currencies - please check the documentation\n",
+    "        }\n",
+    "    }\n",
+    "    response = requests.post(url,headers=headers,json=data) # this establishes connection to the API\n",
+    "    response_data = json.loads(response.text) # this line queries the data\n",
+    "\n",
+    "\n",
+    "pd.json_normalize(response_data) # this line provides a common dataframe out of the API Response\n"
```

Comment:
Suggest we mention that you can print it, or "if you're using `pandas` you can get a dataframe like this" - basically I'd like to ensure that people who might not know what `pd` stands for, or maybe even what pandas is, finds this useful too.

Reply:
Yess!